app/_locales/en/messages.json

```
@@ -73,6 +73,9 @@
   "accountName": {
     "message": "Account Name"
   },
+  "accountNameDuplicate": {
+    "message": "This account name already exists"
```

Nit: A `description` that explains what this is would be helpful for translators. e.g.

```suggestion
    "message": "This account name already exists",
    "description": "This is an error message shown when the user enters a new account name that matches an existing account name"
```

---

Fixed in: d69e6682ef8cce5a6f0d7dd860acabac7ec614e3